diagnostic.md

```diff
@@ -6,70 +6,92 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
```

Comment:
These will work but they're far more than what is needed.  In `config/routes.rb`:
```ruby
get '/books/:id', controller: 'books', action: 'show'
```
or
```ruby
get '/books/:id', to: 'books#show'
```
or
```ruby
get '/books/:id' => 'books#show'
```